README.md

```diff
@@ -49,7 +49,7 @@ To get a guided experience through the available options, simply run:
 ig
 ```
 
-Upon creation, projects will be automatically served on port 3000 and can be accessed via http://localhost:3000/.
+Upon creation, projects will be automatically served on port 4200 and can be accessed via http://localhost:4200/.
```

Comment:
The port can vary depending on the project type. We can use the paragraph from the wiki page. 
For jQuery projects, navigate to http://localhost:3000/ . The app will automatically reload if you change any of the source files.
Note: For Angular projects you need to navigate to http://localhost:3001/ and for React - http://localhost:3002/

And add the configuration for port 4200 which is the default for ignite ui angular project type.

Reply:
Yeah, that sounds better.